### ISSUE TITLE
Hands crushed in release but ok in editor

### ISSUE THREAD

**Issue by juangea** [leapmotion/LeapUnreal#19]
HI there.

I have a big problem, I was just about to release the app for my client and when I did the packaging for my client and tested it the skin of the hands was completely broken, the actions were functioning correctly, but the mesh whas broken, like crushed with mixed skin weights or something similar.

Broken in package:
![hand_broken_in _package](https://user-images.githubusercontent.com/20524718/46295298-fb33a700-c597-11e8-9f9c-76f1a2ae4468.JPG)

Hand in editor:
![hand_ok](https://user-images.githubusercontent.com/20524718/46295299-fbcc3d80-c597-11e8-8786-cb7253df3699.JPG)

Any idea on how to solve it? I need to release this ASAP

**Comment by juangea** [comment]
I can´t downgrade the full project to 4.19!

It´s completely impossible...I´ll try, but I think I won´t be able, it´s a big project with LOTS of things

**Comment by juangea** [comment]
Thanks for that option, I´ll try it, i´m also trying the downgrade path just in case, but the leap plugin refuses to load, should I have to download a previous plugin version?

Thanks

**Comment by juangea** [comment]
I´m trying to change the animation blueprint from BSLowPolyRightAnimBP to LeapLowPolyRightAnimBP but it´s not working at all, the hand is not showing up

**Comment by juangea** [comment]
And now something weird happened, my right hand is totally broken, even when I have the BS animation blueprint applied and compairing left to right, everything is the same configuration : P

Please help me, it feels like the project is falling apart : P

**Comment by getnamo** [comment]
You can pretty easily delete the plugin and grab a new download in case your assets have changed, optionally make sure you have a backup before doing drastic changes like engine downgrades.

I just checked the lowpoly direct rigging in packaged mode and it appears to have the same issue unfortunately. This is surprising and may help narrow the reason for the bug. It's some behavior only happening in packaged mode which is adding some HMD based rotation early in the data chain. Working through it atm will ping an update if I find something.

**Comment by juangea** [comment]
Yes please, the app should go live 12 october in a big event, the downgrade is completely impossible, I need to be able to compile and package it : S

**Comment by juangea** [comment]
BTW I solved the problem with the right hand, I´m unsure what happened, but I restores a previous version of the files form my SVN repository and everything went back to normal.

Anyways, this is a BIG problem, I hope you can fix it soon please :P

**Comment by juangea** [comment]
BTW is it working with other hands in the demos? Like the high resolution ones or the character hands?

**Comment by getnamo** [comment]
fixed in https://github.com/leapmotion/LeapUnreal/releases/tag/v3.0.1

**Comment by juangea** [comment]
Thanks Getnamo!!!! You don´t know how much you saved my life! :D
